realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/sync/SessionTests.kt

```diff
@@ -502,5 +502,18 @@ class SessionTests {
     fun getOrCreateSession() {
         assertNotNull(app.sync.getOrCreateSession(configuration))
     }
+
+    @Test
+    fun getAllSessions(){
+        val realm = Realm.getInstance(configuration)
+        val session = realm.syncSession
+        val sessions = app.sync.allSessions
+
+        assertNotNull(sessions)
+        assertEquals(1, sessions.size)
+
+        realm.close()
+        session.stop()
```

Comment:
No need to manually stop the session. That should happen when closing the Realm

Reply:
Ok!